cfgov/v1/management/commands/invalidate_all_pages.py

```diff
@@ -0,0 +1,38 @@
+import json
+import logging
+import os
+import requests
+
+from akamai.edgegrid import EdgeGridAuth
+
+from django.core.management.base import BaseCommand
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        auth = EdgeGridAuth(
```

Comment:
Does it make sense to try to somehow unify this with your [`v1.models.akamai_backend.AkamaiBackend`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/akamai_backend.py#L11) class, since it shares some of the same parameters/usage? Like maybe adding a `purge_all` method to that class? Not a big deal, but might be nice to avoid any duplication.